index.html

```diff
@@ -66,13 +70,21 @@ <h1 id="lets_connect">LET'S CONNECT</h1>
     <section class="works">
         <div class="card">
             <div class="snapshoot_wapper">
-                <img class="snapshoot" src="images/Snapshoot Portfolio.png" alt="">
+                <img class="snapshoot mobile" src="./images/Snapshootmbl1.png" alt="">
+                <img class="snapshoot desktop" src="./images/Snapshootdsktp1.png" alt="">
```

Comment:
- [ ] [OPTIONAL] I have noticed at screen size `768px` and slightly beyond, the `year` goes beyond the width of the card 😞 . Kindly consider increasing the width of the cad to make sure that the year does not cross over.

The design 😎 
![passing](https://user-images.githubusercontent.com/99394374/213867382-12737e52-4d99-45e8-8e4f-60aba7415d44.PNG)